README.md

```diff
@@ -106,8 +106,7 @@ Upgrading
 
 ```sh
 # brew
-brew update
-brew reinstall navi
+brew upgrade
```

Comment:
This will upgrade all packages, right?
That's not necessary

Wouldn't `brew upgrade navi` be better?

Reply:
Huh. You're right. Now fixed.